frontend/__tests__/components/operator-lifecycle-manager/clusterserviceversion.spec.tsx

```diff
@@ -88,7 +88,7 @@ describe(ClusterServiceVersionRow.displayName, () => {
   it('renders column for app status', () => {
     const col = wrapper.find('.row').childAt(3);
 
-    expect(col.childAt(0).text()).toEqual(CSVConditionReason.CSVReasonInstallSuccessful);
+    expect(col.childAt(0).text()).toEqual(` ${CSVConditionReason.CSVReasonInstallSuccessful}`);
```

Comment:
Does this work?

```suggestion
    expect(col.childAt(0).text()).toEqual(expect.stringContaining(CSVConditionReason.CSVReasonInstallSuccessful));
```

Reply:
I don't think we need to change the test here if I add the co-icon-and-text class.